pages/linux/nitrogen.md

```diff
@@ -0,0 +1,16 @@
+# nitrogen
+
+> Nitrogen is a fast and lightweight desktop background browser and setter for X Window.
+> More information: <https://github.com/l3ib/nitrogen/>.
+
+- View and set the desired wallpaper/s from a specific directory:
+
+`nitrogen {{path/to/directory/}}`
+
+- Set the wallpaper with automatic size settings:
+
+`nitrogen --set-auto {{path/to/file}}`
+
+- Restore the previous wallpaper:
+
+`nitrogen --restore &`
```

Comment:
The `&` is to send the command to the background? it's not part of the app params itself.
```suggestion
`nitrogen --restore`
```